src/main/java/io/seqera/tower/cli/commands/labels/DeleteLabelsCmd.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2021, Seqera Labs.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * This Source Code Form is "Incompatible With Secondary Licenses", as
+ * defined by the Mozilla Public License, v. 2.0.
+ */
+
+package io.seqera.tower.cli.commands.labels;
+
+import io.seqera.tower.ApiException;
+import io.seqera.tower.cli.commands.computeenvs.AbstractComputeEnvCmd;
+import io.seqera.tower.cli.exceptions.ShowUsageException;
+import io.seqera.tower.cli.exceptions.TowerException;
+import io.seqera.tower.cli.responses.Response;
+import picocli.CommandLine;
+
+import javax.annotation.Nullable;
+import java.io.IOException;
+
+@CommandLine.Command(
+        name = "delete",
+        description = "Delete label"
+)
+public class DeleteLabelsCmd extends AbstractLabelsCmd {
+
+    @CommandLine.Option(names = {"-i", "--id"}, description = "Label ID", required = true)
+    public Long labelId;
+
+    @CommandLine.Option(names = {"-w", "--workspace"}, description = "Workspace numeric identifier", required = false)
+    @Nullable
+    public Long workspaceId;
+
+    @Override
+    protected Response exec() throws ApiException, IOException {
+        try {
+
+            api().deleteLabel(labelId, workspaceId);
+
+        } catch (Exception e) {
+            throw new TowerException(String.format("Unable to delete label '%d', reason: %s", labelId, e.toString()));
+        }
+
+        return new GenericStrResponse(String.format("Label '%d' deleted successfully\n", labelId));
```

Comment:
Remember that with the option `--output json` the response will be serialized to JSON and show as result. This is why a generic response is not a good idea

See a similar example here http://github.com/seqeralabs/tower-cli/blob/70c30c586a05cfc44819aa0cfd2204291f95e7b4/src/main/java/io/seqera/tower/cli/responses/secrets/SecretDeleted.java#L17-L17